src/gateways/clienteGateway.ts

```diff
@@ -1,38 +1,71 @@
+import { and, eq, or, sql } from "drizzle-orm";
+import { PostgresDB } from "external/postgres";
 import { Cliente } from "entities/cliente";
-
-import { ClienteModel } from "../external/mongo/models";
-import { ClienteGateway } from "interfaces/gateways/clienteGateway.interface";
+import { ClienteGateway } from "interfaces/gateways";
 import { ClienteMapper } from "adapters/mappers";
+import { ClienteSchema } from "external/postgres/schemas";
 
-export class ClienteMongoGateway implements ClienteGateway {
-    constructor(private readonly clienteModel: typeof ClienteModel) {}
+export class ClientePostgresGateway implements ClienteGateway {
+    constructor(
+        private readonly postgresDB: typeof PostgresDB,
+        private readonly clienteSchema: typeof ClienteSchema,
+    ) {}
 
     public async create(cliente: Cliente): Promise<Cliente> {
-        const result = await this.clienteModel.create({
-            nome: cliente.nome,
-            email: cliente.email.value,
-            cpf: cliente.cpf.value,
-        });
+        const [result] = await this.postgresDB
+            .insert(ClienteSchema)
```

Comment:
Aqui acho que acabou passando e não puxamos o schema do que recebemos no construtor:

```suggestion
            .insert(this.clienteSchema)
```